docs: update prerequisite section

watson opened the pull request:
Renamed "Installation Requirements" to "Prerequisites" and attempted to make it more universal and less reliant on non-standard package managers.

graphaelli commented:
This shouldn't be needed once #99 goes in.   That in turn was blocked but I think we can resolve all of that today.

watson commented:
It should be updated to show support for Python 2 sure, but what do you mean by it's not needed?

watson commented:
Ah, is the virtualenv requirement gone as well?

graphaelli commented:
Right, no need to virtualenv for just standing up an environment, only python 2 or 3 stdlib is used. 
 virtualenv + python3 is required for running the apm integration tests, even under docker.

graphaelli commented:
@watson good to close this one?

watson commented:
Lost track of it. I still think the requirements section could be improved a bit, so I tried to address your earlier comments and rebased with master.

I wanted to get the reference to installing docker with Homebrew out and instead link to the official Docker install docs. If there was a reason why we showed the brew instructions let me know 😃 

I also added a section on how to install Python (but left the note about the Python versions)

What do you think?

graphaelli commented on README.md:
````diff
@@ -4,20 +4,30 @@ This repo contains tools for end to end (eg agent -> apm server -> elasticsearch
+  ```sh
+  brew install python
+  ```
+- Linux
````
This should be something like `Debian/Ubuntu`

watson replied:
Good point. Fixed :)
